lib/commands/file-movement.js

```diff
@@ -381,6 +448,27 @@ commands.pullFile = async function pullFile (remotePath) {
     : await pullFromRealDevice(this.opts.device, remotePath, true);
 };
 
+commands.mobileDeleteFolder = async function mobileDeleteFolder (opts = {}) {
+  let {remotePath} = opts;
+  if (!remotePath.endsWith('/')) {
+    remotePath = `${remotePath}/`;
+  }
+  return this.isSimulator()
+    ? await deleteFromSimulator(this.opts.device, remotePath)
+    : await deleteFromRealDevice(this.opts.device, remotePath);
+};
+
+commands.mobileDeleteFile = async function mobileDeleteFile (opts = {}) {
```

Comment:
This method has a lot of common stuff with the previous one. How about extracting the common code into a common method call?